documentation/book/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -10,12 +10,17 @@ This procedure describes how to install your own CA certificates and private key
 .Prerequisites
 
 * The Cluster Operator is running.
-* A `Kafka` resource within {ProductPlatformName}
-* Your own X.509 certificates and keys in PEM format for the cluster CA or clients CA. For example, these could be generated by `openssl`, using a command such as:
+* A Kafka cluster is not yet deployed
+* Your own X.509 certificates and keys in PEM format for the cluster CA or clients CA.
 +
-[source,shell,subs="+quotes"]
-openssl req -x509 -new -days _<validity>_ --nodes -out ca.crt -keyout ca.key
-
+** If you want to use cluster or clients CA which is not a Root CA, you have to include the whole chain in the certificate file.
+The chain should be in the following order:
++
+1. The cluster or clients CA
+2. One or more intermediate CAs
+3. The root CA
++
+** All CAs in the chain should be marked as CA in the Basic Constraints extension.
```

Comment:
As far as I understand, the basic constraints extension is a list of constraints so we could say.
I would change "should be marked ... " in "should have CA ... " to be clear that they could have something more for other purposes but they need CA for Strimzi. Using "marked" sounds to me that they have to have just CA in the extension.
Maybe it's just me :-)

Reply:
I rephrased it a bit. I hope it works better for you now.